libs/pChart2.1.3/class/pImage.class.php

```diff
@@ -166,6 +166,17 @@ function stroke($BrowserExpire=FALSE)
      header('Content-type: image/png');
      imagepng($this->Picture);
     }
+    
+   /* Updated code to send image as encoded data */
+   function strokeAsImageData()
```

Comment:
Hi Thomas, I am very unsure about this. Could you put some sample code and clarify me how we could get it done better without getting affected by an update from pchart's library?